src/test/java/uk/gov/hmcts/juror/api/moj/report/standard/PoolAnalysisReportTest.java

```diff
@@ -0,0 +1,254 @@
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+class PoolAnalysisReportTest extends AbstractStandardReportTestSupport<PoolAnalysisReport> {
+
+    private MockedStatic<SecurityUtil> securityUtilMockedStatic;
+    private CourtLocationRepository courtLocationRepository;
+
+    private final LocalDate jan15th = LocalDate.of(2024, 1, 15);
+    private final LocalDate jan16th = LocalDate.of(2024, 1, 16);
+
+    public PoolAnalysisReportTest() {
+        super(
+            QJurorPool.jurorPool,
+            PoolAnalysisReport.RequestValidator.class,
+            DataType.POOL_NUMBER_BY_JP,
+            DataType.SERVICE_START_DATE,
+            DataType.JURORS_SUMMONED_TOTAL,
+            DataType.RESPONDED_TOTAL,
+            DataType.ATTENDED_TOTAL,
+            DataType.PANEL_TOTAL,
+            DataType.JUROR_TOTAL,
+            DataType.EXCUSED_TOTAL,
+            DataType.DISQUALIFIED_TOTAL,
+            DataType.DEFERRED_TOTAL,
+            DataType.REASSIGNED_TOTAL,
+            DataType.UNDELIVERABLE_TOTAL,
+            DataType.TRANSFERRED_TOTAL,
+            DataType.FAILED_TO_ATTEND_TOTAL
+        );
+
+        setHasPoolRepository(false);
+    }
+
+    @BeforeEach
+    @Override
+    public void beforeEach() {
+        this.securityUtilMockedStatic = mockStatic(SecurityUtil.class);
+        this.courtLocationRepository = mock(CourtLocationRepository.class);
+        super.beforeEach();
+    }
+
+    @AfterEach
+    void afterEach() {
+        securityUtilMockedStatic.close();
+    }
+
+    @Override
+    public PoolAnalysisReport createReport(PoolRequestRepository poolRequestRepository) {
+        return new PoolAnalysisReport(this.courtLocationRepository);
+    }
+
+    @Override
+    protected StandardReportRequest getValidRequest() {
+        return StandardReportRequest.builder()
+            .reportType("PoolAnalysisReport")
+            .fromDate(LocalDate.of(2024, 1, 15))
+            .toDate(LocalDate.of(2024, 1, 16))
+            .build();
+    }
+
+    @Override
+    @DisplayName("positivePreProcessQueryTypicalCourtOwner")
+    public void positivePreProcessQueryTypical(JPAQuery<Tuple> query, StandardReportRequest request) {
+        securityUtilMockedStatic.when(SecurityUtil::getActiveOwner).thenReturn(TestConstants.VALID_COURT_LOCATION);
+        report.preProcessQuery(query, request);
+
+        verify(query).where(QPoolRequest.poolRequest.returnDate.between(jan15th, jan16th));
+        verify(query).where(QPoolRequest.poolRequest.owner.eq(SecurityUtil.getActiveOwner()));
```

Comment:
Should verify against the expected value TestConstants.VALID_COURT_LOCATION instead of the value returned from a stub.

Reply:
updated

Reply:
Verify seems to still be using the mock to check against:
<img width="614" alt="image" src="https://github.com/hmcts/juror-api/assets/147524406/73fc0d93-46e4-48c0-aa51-ac2ea1d14ab4">